### PR TITLE
feat(cli): Add `icon` command (tauricon)

### PR DESCRIPTION
**Pull request by FabianLars**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary -> tauricon.js is breaking apart rn.

### Other Information
I'll add some more info here after i got some sleep.

### CODE REVIEW

File: tooling/cli/Cargo.toml [hunk 1]
```diff
@@ -64,6 +64,8 @@ ignore = "0.4"
 ctrlc = "3.2"
 log = { version = "0.4.17", features = [ "kv_unstable", "kv_unstable_std" ] }
 env_logger = "0.9.0"
+icns = { git = "https://github.com/FabianLars/rust-icns", rev = "bc2e492328cc5415a23a5eb6ec25632e8326f7bd" }
```
Comment (lucasfernog) [inline tooling/cli/Cargo.toml]: We'll need to either publish this crate or move it to a module inside cli.rs

Comment (FabianLars) [reply tooling/cli/Cargo.toml]: Yes absolutely, that was mainly to show chip (and you) what I was talking about.
Aaand as discussed on discord I'm not really sure how to proceed with the crate (not happy with it for compressed pngs etc, but current state would be fine for 1.1)

Comment (lucasfernog) [reply tooling/cli/Cargo.toml]: Let's go with `fine for 1.1` for now :P

Comment (FabianLars) [reply tooling/cli/Cargo.toml]: I guess you'd prefer to have that crate part of the tauri org? If so i'd just move my fork to the org and let you handle the publishing 😂

Comment (lucasfernog) [reply tooling/cli/Cargo.toml]: Yeah :D

**Comment by FabianLars** [comment]
Marked as ready for review as i don't have any changes for 1.1 planned anymore.

Only the rust-icns crate release is missing, and some testing on macos would be nice.

_Edit:_ Oh and there's a TODO left about the input size, but we specified 1240 as a requirement for so long, i don't think it's really a problem to keep that for now, and it's not a _hard_ requirement anyway, other sizes will work fine too🤷

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**